core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -127,49 +131,260 @@ void editTariffService() throws Exception {
+        ServiceDto dto = ModelUtils.getServiceDto();
+        String requestedJson = new ObjectMapper().writeValueAsString(dto);
+        String uuid = UUID.randomUUID().toString();
+
+        when(userRemoteClient.findUuidByEmail(principal.getName())).thenReturn(uuid);
+        when(superAdminService.editService(any(ServiceDto.class), anyString()))
+            .thenThrow(new NotFoundException(ErrorMessage.EMPLOYEE_WITH_UUID_NOT_FOUND));
+
+        mockMvc.perform(put(ubsLink + "/editService")
+            .principal(principal)
+            .param("uuid", uuid)
+            .content(requestedJson)
+            .contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isNotFound())
+            .andExpect(result -> assertTrue(result.getResolvedException() instanceof NotFoundException))
+            .andExpect(result -> assertEquals(ErrorMessage.EMPLOYEE_WITH_UUID_NOT_FOUND,
+                result.getResolvedException().getMessage()));
+
+        verify(userRemoteClient).findUuidByEmail(principal.getName());
+        verify(superAdminService).editService(any(ServiceDto.class), anyString());
+        verifyNoMoreInteractions(superAdminService, userRemoteClient);
     }
 
     @Test
     void deleteService() throws Exception {
-        mockMvc.perform(delete(ubsLink + "/deleteService/" + 1L))
+        mockMvc.perform(delete(ubsLink + "/deleteService/" + 1L)
+            .principal(principal)
+            .param("id", "1L"))
             .andExpect(status().isOk());
+
+        verify(superAdminService, times(1)).deleteService(1L);
+        verifyNoMoreInteractions(superAdminService);
+    }
+
+    @Test
+    void deleteServiceIfServiceNotFoundException() throws Exception {
+        long id = 1L;
+        doThrow(new NotFoundException(ErrorMessage.SERVICE_IS_NOT_FOUND_BY_ID + id))
+            .when(superAdminService).deleteService(id);
+
+        mockMvc.perform(delete(ubsLink + "/deleteService/" + 1L)
+            .principal(principal)
+            .param("id", "1L"))
+            .andExpect(status().isNotFound())
+            .andExpect(result -> assertTrue(result.getResolvedException() instanceof NotFoundException))
+            .andExpect(result -> assertEquals(ErrorMessage.SERVICE_IS_NOT_FOUND_BY_ID + id,
+                result.getResolvedException().getMessage()));
+
+        verify(superAdminService).deleteService(id);
+        verifyNoMoreInteractions(superAdminService);
+    }
+
+    @Test
+    void getLocations() throws Exception {
+        mockMvc.perform(get(ubsLink + "/getLocations")).andExpect(status().isOk());
+
+        verify(superAdminService, times(1)).getAllLocation();
+        verifyNoMoreInteractions(superAdminService);
+    }
+
+    @Test
+    void getActiveLocations() throws Exception {
+        mockMvc.perform(get((ubsLink + "/getActiveLocations"))).andExpect(status().isOk());
+
+        verify(superAdminService, times(1)).getActiveLocations();
```

Comment:
`verify` w/o the second params means one call
` verify(superAdminService, times(1)).getActiveLocations();  `
  is equals to 
` verify(superAdminService).getActiveLocations();`

Reply:
+